### PR TITLE
Zhaodong/camera lidar calibration

### PR DESCRIPTION
**Pull request by FrankZhaodong**

### CODE REVIEW

**Comment by Nathan903** [comment]
pointcloud2_listener.py and ImageSubscriber.py works

File: calibration/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Camera and lidar calibration process
+The PDF file contains the description as well as the pictures of the process. In the src folder are the two ros2 packages (camera_listener and pointcloud_listener).
+
+## Run the camera and lidar driver:
+1. Camera driver: ros2 launch ximea_driver ximea_driver.launch.py
```
Comment (stevenwaslander) [inline calibration/README.md:5]: ros2 command should be in quotes (display as code)

File: calibration/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+5. Change the min and max values in filter.py and run the filter. It should output pcd files named set_{i}.pcd, where i is the set number. Put the pcd files into one folder and make another folder to put all the corresponding images. Don’t forget to rename the images as set_{i}.jpg
+6. Use Lidar Camera Calibrator App in MATLAB to do the extrinsic calibration, we want at least 10 sets of accepted data.
+7. After importing the image and pointcloud sets, change the intrinsics to Use Fixed Intrinsics and import the result from intrinsic calibration. And set Cluster Threshold and Dimension Tolerance to reasonable values. 
+  The Cluster Threshold should be greater if the resolution of pointcloud data is low. Hit Detect once all settings are done.
+8. If some sets are still not accepted, use Select Checkerboard function to manually select the points for checkerboard and redo detection.
+9. Check the reprojection error and reprojected lidar points. The error should be below 5 pixels and ideally below 2-3 pixels. To decrease the error, discard the sets in which the reprojection can’t cover the whole plane.
+10. Import and save the tform
```
Comment (stevenwaslander) [inline calibration/README.md:43]: Can you make these last steps a little more clear?

**Review by stevenwaslander — COMMENTED** [review]
Just some minor comments on readme, the rest looks fine, already tested by Nathan and used to get calibration.

**Review by stevenwaslander — APPROVED** [review]

**Pull request merged**